config/bartworks.cfg

```diff
@@ -112,8 +493,6 @@ system {
 
     # Enables or Disables the debug log.
     B:"Enable Debug Log"=false
-
-    # Enables or Disables the Experimental Threaded Material Loader.
     B:"Enable Experimental Threaded Material Loader"=false
```

Comment:
this can be deleted completely

Reply:
Those lines are already deleted, or do you mean the 'B:"Enable Experimental Threaded Material Loader"=false' line?

Reply:
the 'B:"Enable Experimental Threaded Material Loader"=false' line, cause thats been removed